src/OGL-UK-1.0.xml

```diff
@@ -0,0 +1,65 @@
+            <p>Using Information under this licence</p>
+            <p>Use of copyright and database right material expressly made available under this licence (the 'Information') indicates your acceptance of the terms and conditions below.</p>
+            <p>The Licensor grants you a worldwide, royalty-free, perpetual, non-exclusive licence to use the Information subject to the conditions below.</p>
+            <p>This licence does not affect your freedom under fair dealing or fair use or any other copyright or database right exceptions and limitations.</p>
+            <p>You are free to:</p>
+            <list>
+                <item>copy, publish, distribute and transmit the Information;</item>
+                <item>adapt the Information;</item>
+                <item>exploit the Information commercially for example, by combining it with other Information, or by including it in your own product or application.</item>
+            </list>
+            <p>You must, where you do any of the above:</p>
+            <list>
+                <item>acknowledge the source of the Information by including any attribution statement specified by the Information Provider(s) and, where possible, provide a link to this licence;
+                    <p>  If the Information Provider does not provide a specific attribution statement, or if you are using Information from several Information Providers and multiple attributions are not practical in your product or application, you may consider using the following:
+                        <br/>Contains public sector information licensed under the Open Government Licence v1.0.
+                    </p>
+                </item>
+                <item>ensure that you do not use the Information in a way that suggests any official status or that the Information Provider endorses you or your use of the Information;</item>
+                <item>ensure that you do not mislead others or misrepresent the Information or its source;</item>
+                <item>ensure that your use of the Information does not breach the Data Protection Act 1998 or the Privacy and Electronic Communications (EC Directive) Regulations 2003.</item>
+            </list>
+            <p>These are important conditions of this licence and if you fail to comply with them the rights granted to you under this licence, or any similar licence granted by the Licensor, will end automatically.</p>
+            <p>Exemptions</p>
+            <p>This licence does not cover the use of:</p>
+            <list>
+                <item>personal data in the Information;</item>
+                <item>information that has neither been published nor disclosed under information access legislation (including the Freedom of Information Acts for the UK and Scotland) by or with the consent of the Information Provider;</item>
```

Comment:
Argh, I think the test is failing now b/c of different capitalization -- "information" vs. "Information" for the first word...

Reply:
...which is odd, I thought the matching guidelines said to ignore capitalization. I could be wrong about this one re: why the test is failing.

Reply:
we'll see... changed it. it is a defined term here...